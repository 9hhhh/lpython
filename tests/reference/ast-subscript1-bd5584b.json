{
    "basename": "ast-subscript1-bd5584b",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/subscript1.py",
    "infile_hash": "2046e24d81cbd28eb23052804f256dc73c52bf0b1677d787078607b1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-subscript1-bd5584b.stdout",
    "stdout_hash": "a1bdbcd5217c00a08079072c0580e6fea0a95e94aa73905cd3932ba5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}