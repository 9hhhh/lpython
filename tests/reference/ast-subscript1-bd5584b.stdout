(Module [(FunctionDef test_subscript ([] [] [] [] [] [] []) [(AnnAssign (Name s Store) (Name str Load) () 1) (Assign [(Name s Store)] (ConstantStr "abc" ()) ()) (Assign [(Name s Store)] (Subscript (Name s Load) (ConstantInt 0 ()) Load) ()) (Assign [(Name s Store)] (Subscript (Name s Load) (Slice (ConstantInt 1 ()) (ConstantInt 2 ()) ()) Load) ()) (AnnAssign (Name A Store) (Subscript (Name i32 Load) (ConstantInt 5 ()) Load) () 1) (AnnAssign (Name B Store) (Subscript (Name i32 Load) (ConstantInt 2 ()) Load) () 1) (AnnAssign (Name i Store) (Name i32 Load) () 1) (Assign [(Name i Store)] (Subscript (Name A Load) (ConstantInt 0 ()) Load) ()) (Assign [(Name B Store)] (Subscript (Name A Load) (Slice (ConstantInt 1 ()) (ConstantInt 3 ()) ()) Load) ())] [] () ())] [])
